src/organisms/cards/StaticMapCard/Readme.md

```diff
@@ -4,6 +4,7 @@ StaticMapCard Example:
     <StaticMapCard
       label='A map has no label'
       center='10010'
+      subheader='New York, NY 10011'
```

Comment:
@jmcolella yes in renters we are gonna hit their geocode api to reverse geocode the zip, and then pass that value to `subheader`